src/LayerManager.h

```diff
@@ -0,0 +1,46 @@
+#include <stdlib.h>
+#include <fstream>
+#include <iostream>
+
+#include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/fstream.hpp>
+#include <boost/filesystem.hpp>
+
+#include <Wt/WContainerWidget>
+#include <Wt/WHBoxLayout>
+#include <Wt/WVBoxLayout>
+#include <Wt/WBorderLayout>
+#include <Wt/WPushButton>
+#include <Wt/WGridLayout>
+#include <Wt/WGroupBox>
+#include <Wt/WVBoxLayout>
+#include <Wt/WHBoxLayout>
+#include <Wt/WImage>
+#include <Wt/WCheckBox>
+#include <Wt/WLineEdit>
+#include <Wt/WText>
+#include <Wt/WLabel>
+#include <Wt/WSelectionBox>
+
```

Comment:
same as above. check if all these includes are required. 

put includes in .h and include the class's header inisde the .cpp

in startup.cpp you can include startup.h and all those wt based includes go into startup.h.

remember some of them are required to included in .cpp file also. So check them